daml-foundations/daml-ghc/daml-prim-src/GHC/Types.daml

```diff
@@ -28,21 +28,22 @@ import GHC.Prim
 
 infixr 5 ::
 
--- | The kind of constraints, like `Show a`
+-- | A type for constraints, like `Show a`
```

Comment:
This really is a "kind", not a type. In Haskell you have terms described by types, types described by kinds (and then kinds described by sorts, if you really want to get fancy - but no one does).

Reply:
Thanks, that's exactly the kind (ha ha) of trap I thought I might fall into here.

Should I change all of these to kinds, or does that just apply to this one?

Reply:
Just this one.